pages/DSNP/Types/Reaction.md

```diff
@@ -41,6 +42,22 @@ None of the following should be considered valid:
 ```
 "F", ":custom-emoji:", "<custom-emoji>", "ᚱ", "ᘐ", "״"
 ```
+### apply
+- MUST be a UINT_8
+Indicates whether the emoji should be applied and if so, its "strength".
```

Comment:
I'm not sure how I feel about allowing for a "strength" value to be applied. I like the potential use cases, but I'm not sure what the unintended consequences of allowing it might be.

The 0-255 range might be too much. I wonder about just a 4 bit number? (0-15) Hmm... It feels like we are trying to smash two separate things together.

Reply:
I feel like it leaves some flexibility without actually breaking what a "reaction" is.  0-255 I think is fine, 0-15 is practically fine too... let's keep it 0-255 as your suggestion for now